kuma/javascript/src/search-results-page.jsx

```diff
@@ -45,160 +45,162 @@ function makePaginationPageURL(uri) {
     return uri ? `?${uri.split('?')[1]}` : '';
 }
 
-export default function SearchResultsPage({ locale, query, data }: Props) {
-    // Because it's clunky to do conditionals in JSX, let's create all the
-    // React nodes here first.
-    let resultsMetaNode = null;
-    let resultsNode = null;
-    let noResultsNode = null;
-    let pagerNode = null;
-    let errorNode = null;
-    let noQuery = null;
-
-    if (!query) {
-        noQuery = (
-            <div className="result-container">
-                <p>
-                    <i>{gettext('Nothing found if nothing searched.')}</i>
-                </p>
-            </div>
-        );
-    }
-
-    if (data) {
-        const { results } = data;
+function ResultsMeta({
+    locale,
+    results
+}: {
+    locale: string,
+    results: SearchResults
+}) {
+    return (
+        <div className="result-container">
+            <p className="result-meta">
+                {interpolate(
+                    ngettext(
+                        '%(count)s document found for "%(query)s" in %(locale)s.',
+                        '%(count)s documents found for "%(query)s" in %(locale)s.',
+                        results.count
+                    ),
+                    {
+                        count: results.count.toLocaleString(),
+                        // XXX this 'locale' is something like 'en-US'
+                        // need to turn that into "English (US)".
+                        locale: locale,
+                        query: results.query
+                    }
+                )}{' '}
+                {!!results.count &&
```

Comment:
Can you do the same technique here. So...
```jsx
{!!results.count && (
  <>
  </>
)}
```

Reply:
Also, I think the logic should be:
```
if (results.count) {
  if (results.previous || results.next) {
      "Showing results ..."
  } else {
     "Show all results."
  }
}
```
See how nicer it is with "traditional" if statement blocks :)